_search-plugins/hybrid-search.md

```diff
@@ -1213,3 +1213,193 @@ The response contains scoring information:
 Field | Description
 :--- | :---
 `explanation` | The `explanation` object has three properties: `value`, `description`, and `details`. The `value` property shows the result of the calculation, `description` explains what type of calculation was performed, and `details` shows any subcalculations performed. For score normalization, the information in the `description` property includes the technique used for normalization or combination and the corresponding score. 
+
+## Paginate hybrid query results
+**Introduced 2.19**
+{: .label .label-purple }
+
+You can apply pagination to hybrid query results by using the `pagination_depth` parameter in the hybrid query clause, along with the standard `from` and `size` parameters. The `pagination_depth` parameter defines the maximum number of search results that can be retrieved from each shard per subquery. For example, setting `pagination_depth: 50` allows up to 50 results per subquery to be maintained in memory from each shard.
+
+To navigate through the results, use:
+- `from`: specifies the document number from which you want to start showing the results, default is `0`
+- `size`: specifies the number of results to return on each page, default is `10`
+
+For example, to show results from 20th document to 30th document, set `from: 20` and `size: 10`. For more information about pagination, see [paginate results]({{site.url}}{{site.baseurl}}/search-plugins/searching-data/paginate/#the-from-and-size-parameters).
+
+### The impact of pagination_depth on hybrid search results
```

Comment:
This paragraph is let user know that changing the pagination_depth also changes the search results reference.